gguf-py/gguf/constants.py

```diff
@@ -19,18 +19,29 @@
 
 class Keys:
     class General:
-        ARCHITECTURE         = "general.architecture"
-        QUANTIZATION_VERSION = "general.quantization_version"
-        ALIGNMENT            = "general.alignment"
-        NAME                 = "general.name"
-        AUTHOR               = "general.author"
-        VERSION              = "general.version"
-        URL                  = "general.url"
-        DESCRIPTION          = "general.description"
-        LICENSE              = "general.license"
-        SOURCE_URL           = "general.source.url"
-        SOURCE_HF_REPO       = "general.source.huggingface.repository"
-        FILE_TYPE            = "general.file_type"
+        ARCHITECTURE           = "general.architecture"
+        QUANTIZATION_VERSION   = "general.quantization_version"
+        ALIGNMENT              = "general.alignment"
+        NAME                   = "general.name"
+        BASENAME               = "general.basename"
+        FINETUNE               = "general.finetune"
+        AUTHOR                 = "general.author"
+        QUANTIZED_BY           = "general.quantized_by"
+        ORGANIZATION           = "general.organization"
+        VERSION                = "general.version"
+        BASE_VERSION           = "general.base_version"
+        URL                    = "general.url"
+        DESCRIPTION            = "general.description"
+        LICENSE                = "general.license"
+        LICENSE_NAME           = "general.license.name"
+        LICENSE_LINK           = "general.license.link"
+        SOURCE_URL             = "general.source.url"
+        SOURCE_HF_REPO         = "general.source.huggingface.repository"
+        FILE_TYPE              = "general.file_type"
+        PARAMETER_WEIGHT_CLASS = "general.parameter_weight_class"
```

Comment:
is this an enum?

Reply:
(parsed in `Metadata.get_model_name_components`)

Reply:
No, they're class attributes.